django/river/adapters/progression_counter.py

```diff
@@ -10,6 +10,7 @@
 class Progression:
```

Comment:
I'm fine with it anyway but changes in this file aren't really related to sending mappings to river when creating a batch, right?

Reply:
Nop, Simon made this changes to fix the integration tests

Reply:
Fine 👍